vector/v.lidar.correction/v.lidar.correction.html

```diff
@@ -117,8 +106,12 @@ <h2>REFERENCES</h2>
 <a href="http://www.itc.nl/isprswgIII-3/filtertest/MainDoc.htm">ISPRS WG III/3 Comparison of Filters</a> 
 report by Sithole, G. and Vosselman, G., 2003. 
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+<h2>AUTHORS</h2>
 
+Original version of program in GRASS 5.4:
+<br>
+Maria Antonia Brovelli, Massimiliano Cannata, Ulisse Longoni and Mirko Reguzzoni
+<br><br>
+Update for GRASS 6.X:
+<br>
+Roberto Antolin and Gonzalo Moreno
```

Comment:
Line 55, 68 unnecessary one empty line.

`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.